src/daffodilDebugger.ts

```diff
@@ -120,18 +120,8 @@ export async function getDebugger(
         await unzipFile(filePath, rootPath)
       }
 
-      // Stop debugger if running
-      if (os.platform() === 'win32') {
-        // Windows stop debugger if already running
-        child_process.execSync(
-          'tskill java 2>nul 1>nul || echo "Java not running"'
-        )
-      } else {
-        // Linux/Mac stop debugger if already running and make sure script is executable
-        child_process.exec(
-          "kill -9 $(ps -ef | grep 'daffodil' | grep 'jar' | awk '{ print $2 }') || return 0"
-        ) // ensure debugger server not running and
-      }
+      // stop java program -- in this case the debugger
+      await stopJavaProcess('daffodil')
```

Comment:
Refactoring this as a utility function is a good idea.  Instead of sending in the process _name_, instead send it the process ID to be more targeted and safe about the process killing.